### PR TITLE
Minor Bug Fixes

### PR DESCRIPTION
**Pull request by Blunderchips**
Two minor bug fixes. Changed the access modifies of five classes to from default public and fixed the parameters of two methods.

Change the parameters of the exception logging methods to use Throwable instead of Exception.
ConsoleUtils.exceptionToString already uses Throwables.

Made the following classes public:

1. CommandCompleter.java
2. CommandHistory.java
3. ConsoleContext.java
4. Log.java
5. LogEntry.java

### CODE REVIEW

**Comment by StrongJoshua** [comment]
Why do you think these classes all need to be public?

**Comment by Blunderchips** [comment]
I am attempting to make VisGUIConsole for my current project. I am achieving this by extending the AbstractConsole class and recreating the GUIConsole using visui components. To do this, access to those classes is required. 

Is there any particular reason you want them protected?

The final commit changes a variable type in two of the logging methods. The methods currently log exceptions meaning that errors and general throwables can be logged. Your console utilities class already accepts throwables.

**Comment by Blunderchips** [comment]
TL;DR It gives users more control over the console which, for a library, is always good.

**Comment by StrongJoshua** [comment]
Sounds good. Was just curious.
